contracts/teller-network/Escrow.sol

```diff
@@ -69,6 +79,63 @@ contract Escrow is Pausable, MessageSigned, Fees, Arbitrable {
     event Rating(uint indexed offerId, address indexed buyer, uint indexed escrowId, uint rating, uint date);
 
 
+    mapping(address => uint) public lastActivity;
+
+    function accept_relayed_call(address relay, address from, bytes memory encoded_function, uint gas_price,
+      uint transaction_fee ) public view returns(uint32) {
+
+        bytes4 fSign = abiDecodeFunctionSignature(encoded_function);
+
+        if(fSign != CREATE_SIGNATURE && fSign != PAY_SIGNATURE && fSign != CANCEL_SIGNATURE && fSign != OPEN_CASE_SIGNATURE)
+            return 11;
```

Comment:
What happens when this returns not 0 (when it fails)? Does the relayer till pay gas or does it revert?

Reply:
anything different than 0 will not be relayed, and relayer does not pay gas nor anything. I guess they do a .call() first to see what will be the result of the call